### PR TITLE
fix: clearer chompfile

### PR DESCRIPTION
**Pull request by Aslemammad**

### CODE REVIEW

File: chompfile.toml [hunk 1]
```diff
@@ -14,9 +14,9 @@ run = 'cargo build --release --locked'
 
 [[task]]
 name = 'test'
-dep = 'build'
+deps = ['build', 'install']
 cwd = 'test'
-run = '../target/debug/chomp test'
+run = 'chomp test'
```
Comment (guybedford) [inline chompfile.toml:19]: We need to ensure that the tests run against the local Chomp version though.

File: test/chompfile.toml [hunk 2]
```diff
@@ -29,7 +29,7 @@ deps = [':test[0-9]*']
 [[task]]
 name = 'test1'
 display = 'none'
-run = '../target/debug/chomp output/test1.txt'
+run = 'chomp output/test1.txt'
```
Comment (guybedford) [inline test/chompfile.toml:32]: Again, this would use the global chomp instead of the local version for testing.

File: chompfile.toml (hunk 1 above)
Comment (Aslemammad) [reply chompfile.toml:19]: After install task, wouldn't be the global chomp the new debugging build? So it's always updated and it's the `target/debug/chomp`!

Let me know if I'm wrong.

File: chompfile.toml [hunk 3]
```diff
@@ -14,9 +14,9 @@ run = 'cargo build --release --locked'
 
 [[task]]
 name = 'test'
-dep = 'build'
+deps = ['build', 'install']
```
Comment (Aslemammad) [inline chompfile.toml:17]: And here, it would always run after build.

Comment (Aslemammad) [inline chompfile.toml:17]: And here, it would always run after build.

**Comment by guybedford** [comment]
The install task doesn’t run by default though - we don’t assume the
testing environment will run it.

**Comment by Aslemammad** [comment]
I added it to the `test` deps, still it won't run?

**Comment by Aslemammad** [comment]
Anyways, if still this does not make sense, feel free to close it.

**Comment by guybedford** [comment]
Yeah install isn’t always reliable because we don’t have a reliable
directory in the Env path to copy into. It could be cool to better provide
ways to customize env paths per commands to allow this though… I think
there may be some issues on that or if not we could plan some.

**Comment by Aslemammad** [comment]
Aha, that's reasonable! Let's close.

**Pull request closed without merging**